File: src/main/java/io/femo/http/transport/http2/HttpFlowControlException.java
package io.femo.http.transport.http2;

import org.jetbrains.annotations.NonNls;

/**
 * Created by felix on 9/4/16.
 */
public class HttpFlowControlException extends RuntimeException {

    public HttpFlowControlException(@NonNls String message) {
        super(message);
    }

    public HttpFlowControlException(String message, Throwable cause) {
        super(message, cause);
    }

    public HttpFlowControlException(Throwable cause) {
        super(cause);
    }
}

File: src/main/java/org/web25/http/transport/http2/HttpFlowControlException.kt
package org.web25.http.transport.http2

import org.jetbrains.annotations.NonNls

/**
 * Created by felix on 9/4/16.
 */
class HttpFlowControlException : RuntimeException {

    constructor(@NonNls message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(cause: Throwable) : super(cause) {}
}
